includes/Plugin.php

```diff
@@ -149,7 +149,7 @@ function() use ( $options, $transients, $assets ) {
 				$modules->register();
 
 				( new Core\Permissions\Permissions( $this->context, $authentication ) )->register();
-				( new Core\Util\Tracking( $this->context, $authentication ) )->register();
+				( new Core\Util\Tracking( new Core\Util\Tracking_Consent( $user_options ) ) )->register();
```

Comment:
See my other comment above. If we pass this here, we should also pass all the other option/user option objects to `Authentication`. Which is up for discussion, but it would maybe lead us down a rabbit hole (tons of parameters!). That's why I think we should not use DI here for now.